condor_watch_q.py

```diff
@@ -435,7 +435,7 @@ def watch_q(
         exit_checks.append((exit_grouper, exit_check, exit_code, disp))
 
     try:
-        msg = None
+        msg, move, clear = None, None, None
 
         while True:
             with display_temporary_message("Reading new events...", enabled=refresh):
```

Comment:
Although it's technically a lie and I hate how it's going to affect the indent, I think we should probably lie and put everything up until we print the clear under this `with`. Maybe change it to "Processing new events...". Then we should always have something on the screen (the particular flickering problem we observed was also partially that it took a long time to make the table, not just to read new events).